content/en/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Configuring CA Certificates
```

Comment:
```suggestion
title: Plugging in existing CA Certificates
```

Reply:
"Configuring" is very unclear. It could mean "configuring a self-signed cert" which is definitely not what you trying to say.

Reply:
Done.